addon/services/raven.js

```diff
@@ -4,7 +4,7 @@ export default RavenService.extend({
     releaseMetaName: 'sentry:revision',
     release: Ember.computed({
```

Comment:
Updated with dependent key and squashed. I also imported Ember and updated to user Ember.$ instead of jQuery directly. Let me know if you disagree and I'll remove the Ember import and switch back to using jQuery directly :)